templates/operations.ts.hbs

```diff
@@ -122,19 +122,24 @@
       const optionParams = options?.parameters || {} as UrlParameters;
       const configParams = this.clientConfig.parameters || {};
 
+      // All config parameters are assumed to be path parameters
       const pathParams = {
+        version: 'v1',
+        ...configParams
+      } as PathParameters;
+      // Merge in known path parameters from options
       {{#each ../parameters}}
-        "{{{name}}}": optionParams["{{{name}}}"]
-        {{! if it is a common path parameter, fall back to client config }}
-        {{#if (isCommonPathParameter name)}}  ?? configParams["{{{name}}}"]{{/if}}
-        {{#unless @last}},{{/unless}}
+      if (optionParams["{{{name}}}"] !== undefined) {
```

Comment:
As this is a bug fix where is the accompanying test proving that it fixes the issue?